### PR TITLE
added movieposters on moviepage with styling, added one more movie in…

### PR DESCRIPTION
**Pull request by mhema42**
… DB, added animation for image in mainhero

image

### CODE REVIEW

**Review by emmaambr — APPROVED** [review]
Looks good! Really like the fade-in animation 💯

**Review by simonlj8 — APPROVED** [review]
Good job! Nice with fade-in

File: components/layout/MainHero.module.css [hunk 1]
```diff
@@ -18,4 +18,18 @@
...
+}
+
+.image {
+    animation: fade-in 1s forwards 1;
+  }
+  
+  @keyframes fade-in {
+    0% {
+      opacity: 0;
+    }
```
Comment (richardolsson) [inline components/layout/MainHero.module.css:30]: The indentation here is not very consistent. If you find it difficult to keep it consistent by hand, consider activating automatic formatting in your editor.

File: components/movies/MovieLink.module.css [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
-    margin: 1rem;
+.image img {
+  padding: 2rem;
+  width: 100%;
+  height: 25vw;
+  object-fit: contain;
+}
+
+.image img:hover {
+    cursor: pointer;
```
Comment (richardolsson) [inline components/movies/MovieLink.module.css:9]: Another case of inconsistent formatting.

File: pages/movies/index.js [hunk 3]
```diff
@@ -11,12 +22,12 @@ export async function getStaticProps() {
         props: {
             posters: posters.map((poster) => ({
                 title: poster.title,
-                id: poster._id.toString()
+                poster: poster.poster,
+                id: poster._id.toString(),
             })),
         },
-        revalidate: 1
+        revalidate: 1,
```
Comment (richardolsson) [inline pages/movies/index.js:29]: To revalidate every second makes little sense. It means content is only cached for one second, and you wouldn't really notice a difference unless you have significantly more than one user per second. For less than one user per second you might actually see performance _degrade_ as generating the static page (and writing it to cache) might take slightly longer than generating a dynamic one on-demand.

**Review by richardolsson — COMMENTED** [review]
Nice work, even though a large portion of it was cosmetic changes. I have left some comments below for future reference.

**Pull request merged**